bootstrap

```diff
@@ -39,8 +39,8 @@ virtualenv --no-site-packages --distribute virtualenv
 # avoid pip bugs
 ./virtualenv/bin/pip install --upgrade pip
 
-# slightly old version of setuptools; newer fails w/ requests 0.14.0
-./virtualenv/bin/pip install setuptools==32.3.1
+# latest setuptools supporting python 2.7
+./virtualenv/bin/pip install setuptools==44.1.0
```

Comment:
pin a version compatible with python2 so the test won't fail.

because, if setuptool does not support python2 and the backport of https://github.com/ceph/ceph/pull/35226 is not merged yet, we will be using a venv of py2 to install setuptools only supporting py3.